simulator/risc_v/riscv_instr.cpp

```diff
@@ -287,6 +288,8 @@ static const std::vector<RISCVTableEntry<I>> cmd_desc =
     /*-------------- B --------------*/
     {'B', instr_slo,      execute_slo<I>,  OUT_ARITHM, ' ', Imm::NO,    Src1::RS1,  Src2::RS2,  Dst::RD,   0, 32 | 64      },
     {'B', instr_orn,      execute_orn<I>,  OUT_ARITHM, ' ', Imm::NO,    Src1::RS1,  Src2::RS2,  Dst::RD,   0, 32 | 64      },
+    // Bit-Field
```

Comment:
What is the purpose of that comment?

Reply:
To mark alu for bit-field instructions. I can remove it.

Reply:
Then it's better to put it after OUT_ARITHM.

Reply:
Didn't understand what place do you mean

Reply:
`OUT_ARITHM` defines the type of ALU used in performance simulator for cycle-accurate modeling. Bit-manipulation ALU has different timing, and, probaly, we need another enumeration value here - do I understand the intention of your initial comment correctly?

(I just try to reduce amount of unnecessary comments. Comment should be a "warning sign" that something goes in unusual way here. If there are too many comments, people will just ignore them)

Reply:
Probably it makes no sense to add a separate comment for just two instructions.